src/_data/weeklies.js

```diff
@@ -0,0 +1,14 @@
+const fs = require("fs");
+const path = require("path");
+
+const weekliesFolder = path.resolve(__dirname, "../events/weeklies");
+
+const movies = fs
+  .readdirSync(weekliesFolder)
+  .filter((name) => path.extname(name) === ".json")
+  .map((name) => ({
+    key: path.parse(name).name,
+    ...require(path.join(weekliesFolder, name)),
+  }));
+
+module.exports = movies;
```

Comment:
this file is how 11ty knows about the weeklies dataset. lets us keep them in separate files managed by the CMS.